wp-content/themes/core/routes/unsupported_browser/unsupported_browser.php

```diff
@@ -28,19 +28,31 @@
 
 	<div class="site-header">
 		<div class="l-container">
-			<h1 class="site-brand">
-				<img src="<?php echo $c->get_legacy_image_url( 'logo-header.png' ); ?>"
-					 class="site-logo site-logo--header"
-					 alt="<?php printf( '%s %s', esc_attr( get_bloginfo( 'name' ) ), esc_attr( __( 'logo', 'tribe' ) ) ); ?>" />
-			</h1>
+			<div class="g-row g-row--col-1">
+				<div class="g-col site-header--underline">
```

Comment:
In using BEM `site-header--underline` would be a modifying class that would go on the same container as where `site-header` is. What you need is an child element implementation `site-header__underline`

Reply:
✅